app/assets/stylesheets/active_admin_sidebar.scss

```diff
@@ -0,0 +1,87 @@
+@import "active_admin_sidebar_pure_icons";
+
+body.active_admin {
+  #active_admin_content.left_sidebar, #active_admin_content.collapsed_sidebar {
```

Comment:
Avoid using id selectors<br>Selector `active_admin_content` should be written in lowercase with hyphens<br>Selector `left_sidebar` should be written in lowercase with hyphens<br>Selector `collapsed_sidebar` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line